src/highdicom/sr/templates.py

```diff
@@ -3721,6 +3847,16 @@ def __init__(
             )
         )
         self.append(pixel_data_cols_item)
+
+        if self._is_cross_sectional(image):
```

Comment:
Here, we may want to add additional descriptors for SM images (e.g., Total Pixel Matrix Rows/Columns). However, this is not yet part of the standard and we can address this later.

Reply:
Maybe this should be addressed with changes needed for EnhancedMR?